tests/packagedcode/data/podspec/nanopb.podspec.expected.json

```diff
@@ -0,0 +1,45 @@
+{
+  "type": "pods",
+  "namespace": null,
+  "name": "nanopb",
+  "version": "1.30905.0",
+  "qualifiers": {},
+  "subpath": null,
+  "primary_language": "Objective-C",
+  "description": " Nanopb is a small code-size Protocol Buffers implementation Nanopb is a small code-size Protocol Buffers implementation in ansi C. It is especially suitable for use in Nanopb is a small code-size Protocol Buffers implementation Nanopb is a small code-size Protocol Buffers implementation in ansi C. It is especially suitable for use in microcontrollers, but fits any memory restricted system.",
+  "release_date": null,
+  "parties": [
+    {
+      "type": "person",
+      "role": "author",
+      "name": "Petteri Aimonen",
+      "email": "jpa@nanopb.mail.kapsi.fi",
+      "url": null
+    }
+  ],
+  "keywords": [],
+  "homepage_url": "https://github.com/nanopb/nanopb",
+  "download_url": null,
+  "size": null,
+  "sha1": null,
+  "md5": null,
+  "sha256": null,
+  "sha512": null,
+  "bug_tracking_url": null,
+  "code_view_url": null,
+  "vcs_url": "https://github.com/nanopb/nanopb.git",
+  "copyright": null,
+  "license_expression": "unknown",
```

Comment:
We definitely need to override the license detection in order to detect correctly these declared_license 
Also is there a way to collect ALL the license values of ALL the podspecs? Ask around for help for this or check in https://github.com/CocoaPods/Specs ... this will be useful to properly create mapping of they license conventions to our keys (in addition to applying proper detection to their license field structure)
That should be in its own ticket though, so create one